docs/code-snippets/outlook-snippets.yaml

```diff
@@ -505,6 +587,52 @@ Office.AppointmentRead.subject:
             // which can be a message or appointment.
         });
     }
+Office.AttachmentContent.format:
+  - |-
+    var item = Office.context.mailbox.item;
+    var listOfAttachments = [];
+    item.getAttachmentsAsync(callback);
+    function callback(result) {
+        if (result.value.length > 0) {
+            for (i = 0 ; i < result.value.length ; i++) {
+                var options = {asyncContext: {type: result.value[i].attachmentType}};
+                getAttachmentContentAsync(result.value[i].id, options, handleAttachmentsCallback);  
+            }
+        }
+    }
+
+    function handleAttachmentsCallback(result) {
+        // parse string to be a url, an .eml file, a base64-encoded string, or an .icalendar file
+        if (result.format == Office.MailboxEnums.AttachmentContentFormat.Base64) {
+            // handle file attachment
+        } else if (result.format == Office.MailboxEnums.AttachmentContentFormat.Eml) {
+            // handle item attachment
+        } else if (result.format == Office.MailboxEnums.AttachmentContentFormat.ICalendar) {
+            // handle .icalender attachment
+        } else {
```

Comment:
If this is another enum value, consider using that and not having a general else case. That way, the sample will be robust and still valid when another AttachmentContentFormat gets adds.